### DISCUSSION TITLE
Do thorough API docs make sense yet?

### DISCUSSION THREAD

**Discussion by Tamschi** [bevyengine/bevy#2336] [Q&A] [answered]
I noticed the API docs are fairly sparse and mostly don't have usage examples/cross references/(in modules) directions towards API entry points. Does it make sense to add that kind of information there yet?

The engine seems to be, features-wise, at a point now where I'd like to try to actually learn to use it, and I wouldn't mind if docs I write in that process get deleted due to ongoing changes, but I'd rather not hinder development by submitting documentation that's too cohesive for the current stage of development.

**Comment by cart** [comment, marked as answer]
Generally more docs are welcome, as long as you're cool with them being thrown out when we change things (as a lot of apis are still in flux). 

A large number of dense / extensive docs that are hard to review / keep correct might be "too much" though. Maybe start with documenting one feature in the style and depth you're thinking of and we can review that / adjust as needed.
